src/Card/CardHeader.js

```diff
@@ -115,6 +115,7 @@ class CardHeader extends Component {
       textStyle,
       title,
       titleStyle,
+      titleColor,
```

Comment:
We try to sort the properties alphabetically, that should be before the `titleStyle` property.
You can fix the eslint issue like this:
```js
titleColor, // eslint-disable-line no-unused-vars
```